CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,4 @@
 - Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)
+- defaultAcceptFormat variable added to config.js. This variable allows the user to control the Accept header value
```

Comment:
```suggestion
- defaultAcceptFormat variable added to config.js (equivalent to LWM2M_DEFAULT_ACCEPT_FORMAT env var). This variable allows the user to control the Accept header value
```

Reply:
Looking to the documentation it seems that `writeFormat` has been also added. A new line for it in CHANGES_NEXT_RELASE would be desirable.

Reply:
No, `writeFormat` was a configurable variable before this PR. I have only added it to the README.md configuration section and to the env var because the variable wasn't described or added. 
There are several config variables not added to the configuration section in README.md.

Reply:
In that case, I understand is a side fix in the documentation. Thanks!

NTC

Reply:
With regards to writeFormat the following could be added (new line):

```
- Add: LWM2M_WRITE_FORMAT as equivalence of config.writeFormat
```